utilix/rundb.py

```diff
@@ -116,6 +129,10 @@ class Token:
     Object handling tokens for runDB API access.
     
     """
+    string = None
```

Comment:
This ` string` is the `token_string` or something that the rundb api gives you to access the rundb api? Perhaps a comment (and/or somewhat more helpful variable name) will help others understand what is going on.

Reply:
Yes it is the token string (I thought of it as Token.string). Modified the name to token_string